microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,88 @@
 
-|  | Server-side | Client-side |  
+You may want to provide different experiences to users based on their browser. For example, if you include steps on how to configure Microsoft Edge or another browser for use with your site, you may want to detect the browser and then show the appropriate content.
+
+Mechanisms for browser detection:
+
+| Mechanism | Server-side | Client-side |  
 |:--- |:--- |:--- | 
 | **User-Agent Client Hints** \(recommended\) | `Sec-CH-UA` HTTPS header | `navigator.userAgentData` JavaScript method |  
 | **User-Agent string** \(legacy\) | `User-Agent` HTTPS header | `navigator.userAgent` JavaScript method |  
 
-Microsoft recommends that you use [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] whenever possible for the following reasons.  
+This article describes the methods Microsoft Edge supports for retrieving user agent information.
+
+## Feature detection
+
+Microsoft recommends [detecting if a feature is supported][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] in your browser whenever possible instead of detecting the browser.
+
+If you must detect browsers, Microsoft recommends using client hints as follows.
+
+## User-Agent Client Hints
+
+Microsoft Edge supports User-Agent Client Hints starting with version 90.
 
-*   Improve code maintainability.  
-*   Reduce code fragility.  
-*   Reduce code breakage from changes to the User-Agent string.  
-    
-When [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable and you must use user agent detection, use the following format to detect Microsoft Edge user agent on Windows and Android.  
+User-Agent Client Hints is a cleaner, more privacy-preserving way to access browser information such as browser name, version number, platform, and more. Soon the User-Agent string will be frozen and deprecated by various browsers. For example, the Chrome Platform Status site describes the change in [Feature: Reduce User-Agent string information][https://www.chromestatus.com/feature/5704553745874944].
 
-## User-Agent Client Hints  
+Employ User-Agent Client Hints when you want to:
+- Determine if new browser activity is from the expected user.
+- Customize hints or instructions if the user is new to this site.
 
-Starting in Microsoft Edge version 90, access browser information in a cleaner, more privacy-preserving way.  
+Do not use User-Agent Client Hints to:
+- Block *unsupported* browsers.
+- Restrict access to features on your site.
 
-### User-Agent Client Hints HTTPS Header  
+For more information, navigate to the at [W3C Community Draft Report | User-Agent Client Hints][https://wicg.github.io/ua-client-hints/].

```

Comment:
I think there's a typo here. I think this should be:

> For more information, navigate to the specification at

Reply:
Oh yes 🤢  ... fixed.